locales/templates/app.pot

```diff
@@ -635,3 +654,37 @@ msgctxt "dialogue_ocr_share_description"
 msgid "Find screenshot-to-text useful? Share it now!"
 msgstr ""
 
+#. String displayed to inform users search is performing
+msgctxt "search_transition_searching"
+msgid "Searching…"
+msgstr ""
+
+msgctxt "search_error_title_noresult"
+msgid "No results found"
+msgstr ""
+
+msgctxt "search_error_content_try"
+msgid "Try another word"
+msgstr ""
+
+#. Parameter %1$d is the quantity of matched search results, i.e. 100 results
+#, c-format
+msgctxt "search_separator_results"
+msgid "%1$d results"
+msgstr ""
+
+#. String displayed to inform users the progress of text recognition.
+#. Parameter %1$d is the quantity of screenshot files, i.e. 100 screenshots
+#. remaining
+#, c-format
+msgctxt "search_transition_progress"
+msgid "%1$d screenshots remaining…"
```

Comment:
Same as above, we can rather have:
"Screenshots remaining: %1$d"

Reply:
Same as above.